src/pages/instances/InstanceList.tsx

```diff
@@ -465,12 +466,12 @@ const InstanceList: FC = () => {
   );
 
   return (
-    <main className="l-main instance-list">
-      <div
-        className={classnames("p-panel", {
-          "has-side-panel": !!panelParams.instance,
-        })}
-      >
+    <CustomLayout
+      mainClassName={classnames("instance-list", {
+        "has-side-panel": Boolean(panelParams.instance),
```

Comment:
I should have mentioned the css matching `has-side-panel` assumes it to be a child of `instance-list`, which changed now. Please fix that by changing the `.has-side-panel` to `&.has-side-panel`

Reply:
Right, good catch.